docker-compose.yml

```diff
@@ -61,7 +61,7 @@ x-server: &base_server_setup
 
 services:
   db:
-    image: postgis/postgis:11-3.1-alpine
+    image: postgis/postgis:12-3.4-alpine
```

Comment:
Hey @szabozoltan69, doing this might break current setup. Mabye we should use different directory instead for the data? or use a volume instead? 
We will also need to specify how to move data from old db version to new one. 
```
./.db-12/pg:/var/lib/postgresql/data
```

Reply:
Great idea! (Maybe I'll use .db/pg-12)
The new db can be done via a dump and a load, like this:
```
pg_dump -O    -c --if-exists -h 127.0.0.1 -U test test > /tmp/testdump.sql
```
Then the load is via a simple \i ...testdump.sql from the new db.

Reply:
Nice... let's add this to PR description as well